src/index.js

```diff
@@ -23,6 +23,7 @@ const ipldEthStateTrie = require('ipld-eth-star').ethStateTrie
 const ipldEthStorageTrie = require('ipld-eth-star').ethStorageTrie
 const ipldEthTx = require('ipld-eth-star').ethTx
 const ipldEthTxTrie = require('ipld-eth-star').ethTxTrie
+const ipldBin = require('./ipldBin')
```

Comment:
@kumavis mind moving this resolver to here https://github.com/ipld/js-ipld-raw ?